tests/functional/utils/helper.py

```diff
@@ -68,3 +65,13 @@ def add_auth_header(headers: Dict[str, str], auth: Optional[Dict[str, str]]):
         headers["Authorization"] = f"{token_type} {access_token}"
 
     return headers
+
+
+def throw_friendly_error(message: str, url: str, status_code: int, response: str, headers: dict) -> Exception:
```

Comment:
Does this get called anywhere?

Reply:
It was, until it wasn't. I'll remove it.